astro/src/pages/articles/feed.xml.js

```diff
@@ -0,0 +1,25 @@
+import { getCollection } from 'astro:content';
+import rss from '@astrojs/rss';
+
+function convertToRSS(post) {
+  return {
+    title: post.data.title || 'uh-oh, no title',
+    pubDate: post.data.updated_date || post.data.publish_date || '2024-01-01',
+    description: post.data.description || 'uh-oh, no desc',
+    // Compute RSS link from post `slug`
+    // This example assumes all posts are rendered as `/blog/[slug]` routes
+    link: `/articles/${post.slug}`,
```

Comment:
Does the slug include the category?

Reply:
great question @mooreds it was the first thing I checked as well 😄 


```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<rss version="2.0">
<channel>
<title>FusionAuth Articles</title>
<description>The FusionAuth articles offer all kinds of insight on software development, authentication and the FusionAuth product.</description>
<link>https://fusionauth.io/</link>
<item>
<title>Avoiding Authentication System Lock-In</title>
<link>https://fusionauth.io/articles/authentication/avoid-lockin</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/avoid-lockin</guid>
<description>Steps you can take to avoid auth system lock-in include: considering portability, limiting usage, insulating your application with a facade, and having a backup plan.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Combining SSO and MFA in FusionAuth</title>
<link>https://fusionauth.io/articles/authentication/combine-sso-mfa-fusionauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/combine-sso-mfa-fusionauth</guid>
<description>In this article, you'll learn how to combine single sign-on with multi-factor authentication in FusionAuth.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Common Authentication Implementation Risks And How To Mitigate Them</title>
<link>https://fusionauth.io/articles/authentication/common-authentication-implementation-risks</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/common-authentication-implementation-risks</guid>
<description>What are risks of implementing authentication, authorization and user management, and how can you mitigate them?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Developer Benefits of Single Sign-On</title>
<link>https://fusionauth.io/articles/authentication/developer-benefits-single-sign-on</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/developer-benefits-single-sign-on</guid>
<description>What is Single Sign-On and how can it help developers build successful applications?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How Passwordless Authentication Works - A Deep Dive</title>
<link>https://fusionauth.io/articles/authentication/how-passwordless-works</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/how-passwordless-works</guid>
<description>Add passwordless authentication to your application with FusionAuth.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How Single Sign-on (SSO) Works And Why You Should Care</title>
<link>https://fusionauth.io/articles/authentication/how-sso-works</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/how-sso-works</guid>
<description>Integrating single sign-on (SSO) into your applications means your users can access all your applications with one set of credentials, such as a username and password, a magic link, or a WebAuthn passkey.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Authentication Articles</title>
<link>https://fusionauth.io/articles/authentication</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication</guid>
<description>Learn about authentication methods, processes, and implementation by reading our articles on the topic.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Why Login Failures Matter</title>
<link>https://fusionauth.io/articles/authentication/login-failures</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/login-failures</guid>
<description>Are you tracking your login failures? Learn about how to track them, why it matters, and what you should do to avoid login failure.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How FusionAuth Simplifies Compliance with MFA Requirements</title>
<link>https://fusionauth.io/articles/authentication/mfa-compliance-fusionauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/mfa-compliance-fusionauth</guid>
<description>Compliance is a broad term, but one thing is for sure, whatever makes it easier is good. Learn how MFA with FusionAuth can ease your compliance journey.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What is Multi-Factor Authentication (MFA) and How it Works</title>
<link>https://fusionauth.io/articles/authentication/multi-factor-authentication</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/multi-factor-authentication</guid>
<description>Learn what MFA is, why it's crucial, the available factors, and when to consider requiring multi-factor authentication for enhanced security.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Security Implications Of Passwordless Authentication</title>
<link>https://fusionauth.io/articles/authentication/passwordless-authentication-security</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/passwordless-authentication-security</guid>
<description>Learn about passkeys and other passwordless authentication.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Types Of Kubernetes Auth</title>
<link>https://fusionauth.io/articles/authentication/types-of-kubernetes-auth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/types-of-kubernetes-auth</guid>
<description>What are the three levels of Kubernetes authentication?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Securing Your User Experience with MFA</title>
<link>https://fusionauth.io/articles/authentication/user-experience-ux-mfa</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/user-experience-ux-mfa</guid>
<description>A good user experience (UX) should be table stakes. This article delves into the decisions and processes you'll want to consider when adding multi-factor authentication (MFA) to your application.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebAuthn Explained</title>
<link>https://fusionauth.io/articles/authentication/webauthn-explained</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/webauthn-explained</guid>
<description>An in-depth look at how WebAuthn can help you add biometric authentication to your website.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Decoding How WebAuthn Works</title>
<link>https://fusionauth.io/articles/authentication/webauthn</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/webauthn</guid>
<description>A deep dive into the standard of WebAuthn, and how passkeys are bringing passwordless authentication to the masses.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Why Passwordless Authentication Matters in the Age of WebAuthn</title>
<link>https://fusionauth.io/articles/authentication/why-passwordless-authentication-matters</link>
<guid isPermaLink="true">https://fusionauth.io/articles/authentication/why-passwordless-authentication-matters</guid>
<description>Uncover the essentials of WebAuthn for secure, passwordless online authentication and its protection against phishing.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Auth Bottleneck Pattern</title>
<link>https://fusionauth.io/articles/ciam/auth-and-the-bottleneck-architecture</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/auth-and-the-bottleneck-architecture</guid>
<description>Why you might use a centralized user management system rather than having individual applications manage their own auth.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Auth Facade Pattern</title>
<link>https://fusionauth.io/articles/ciam/auth-facade-pattern</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/auth-facade-pattern</guid>
<description>Introducing a facade in front of customer identity providers adds implementation flexibility and gives additional control to your applications</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Challenges of CIAM</title>
<link>https://fusionauth.io/articles/ciam/challenges-of-ciam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/challenges-of-ciam</guid>
<description>CIAM is full of security and user management challenges that take experienced developers months to solve. Learn the features and security your CIAM solution should offer.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How Does Customer Identity And Access Management (CIAM) Differ From Identity And Access Management (IAM)?</title>
<link>https://fusionauth.io/articles/ciam/ciam-vs-iam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/ciam-vs-iam</guid>
<description>CIAM and IAM: What are the main differences between CIAM and IAM (often called workforce)?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Demise of Third-Party Cookies and Why You Need to Run Your Own CIAM System</title>
<link>https://fusionauth.io/articles/ciam/demise-of-third-party-cookies-running-own-ciam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/demise-of-third-party-cookies-running-own-ciam</guid>
<description>What is the future of third party cookies and how does that tie into customer identity and access management?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Developer's Guide To GDPR</title>
<link>https://fusionauth.io/articles/ciam/developers-guide-to-gdpr</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/developers-guide-to-gdpr</guid>
<description>A complete guide for developers to ensure their applications are GDPR compliant.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>CIAM Articles</title>
<link>https://fusionauth.io/articles/ciam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam</guid>
<description>Learn about Customer Identity and Access Management (CIAM) by reading our articles on the topic.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Making Sure Your Auth System Can Scale</title>
<link>https://fusionauth.io/articles/ciam/making-sure-your-auth-system-scales</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/making-sure-your-auth-system-scales</guid>
<description>What steps do you need to take to ensure your authentication system can scale?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Unlocking Growth: Why A Low Friction User Signup Process Is Crucial For Your Business</title>
<link>https://fusionauth.io/articles/ciam/unlocking-growth-low-friction-signup-process</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/unlocking-growth-low-friction-signup-process</guid>
<description>What is the overall value of low-friction user signups? This article covers all the ways low-friction signups add value to your application and business.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What is Customer Identity and Access Management (CIAM)?</title>
<link>https://fusionauth.io/articles/ciam/what-is-ciam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/ciam/what-is-ciam</guid>
<description>CIAM solutions help organizations to securely authenticate and authorize customers, protect customer data, and provide a seamless customer experience</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Benefits of Self-Hosting Critical Infrastructure to Reduce Latency in Gaming</title>
<link>https://fusionauth.io/articles/gaming-entertainment/benefits-self-hosting-reduce-latency</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/benefits-self-hosting-reduce-latency</guid>
<description>While esports competitions are high skill cap events, both hobby gamers and elite professionals alike expect optimal network conditions when they play games. A suboptimal network can turn an enjoyable event into an infuriating one that drives players away.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Best Practices For Handling Spiky Registration Traffic For Game Launches</title>
<link>https://fusionauth.io/articles/gaming-entertainment/best-practices-spiky-registration</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/best-practices-spiky-registration</guid>
<description>What are steps you can take to make sure your servers can handle any spikes in registration traffic?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Cross-Platform Gaming Accounts: Why And How?</title>
<link>https://fusionauth.io/articles/gaming-entertainment/cross-platform-game-accounts</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/cross-platform-game-accounts</guid>
<description>You can build a game that deploys to multiple platforms using tools like Unity. But what about user accounts?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Identity Provider Needs For Gaming Or Esports</title>
<link>https://fusionauth.io/articles/gaming-entertainment/gaming-identity-provider-needs</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/gaming-identity-provider-needs</guid>
<description>What are the specific features you need to look for in an identity provider if you’re in the gaming or esports industry?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Gaming & Entertainment Articles</title>
<link>https://fusionauth.io/articles/gaming-entertainment</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment</guid>
<description>Gaming and Entertainment applications need all the functionality of normal applications, but have a few unique requirements.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How The OAuth Device Authorization Grant Can Make Your Users’ Lives Easier</title>
<link>https://fusionauth.io/articles/gaming-entertainment/oauth-device-grant-gaming</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/oauth-device-grant-gaming</guid>
<description>The OAuth Device Authorization grant enables game users to authenticate using a device where they may already be authenticated, or which may have a far superior input user experience.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Securing Gaming User Accounts With Fraud Detection And Suspicious Activity Notification</title>
<link>https://fusionauth.io/articles/gaming-entertainment/securing-game-account</link>
<guid isPermaLink="true">https://fusionauth.io/articles/gaming-entertainment/securing-game-account</guid>
<description>When it comes to gaming, saving progress is an essential job. Gaming user accounts do that and much more. However, just like any other virtual account, gaming accounts need to be secured from fraudulent activity as well. In this guide, learn about the significance of securing gaming accounts and the ways in which it can be done.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How Complete Does Your Authentication Provider Need To Be?</title>
<link>https://fusionauth.io/articles/identity-basics/complete-authentication-system</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/complete-authentication-system</guid>
<description>Learn the various types of authentication systems, advantages and limitations of each, and which ones might suit your application.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Performing Due Diligence On Authentication Vendors</title>
<link>https://fusionauth.io/articles/identity-basics/due-diligence-authentication-vendors</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/due-diligence-authentication-vendors</guid>
<description>What kind of due diligence should you do when evaluating identity as a service vendors?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Identity Basics</title>
<link>https://fusionauth.io/articles/identity-basics</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics</guid>
<description>Identity and access management is complex and can increase your risks if incomplete or wrong. Use this expert advice to help implement secure and efficient identity solutions for your applications.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Magic Links - A Guide to Passwordless Authentication</title>
<link>https://fusionauth.io/articles/identity-basics/magic-links</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/magic-links</guid>
<description>Magic links offer secure, passwordless authentication, enhancing user experience and security. Learn more in and improve your access control today!</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Multi-Tenant vs. Single-Tenant IDaaS Solutions</title>
<link>https://fusionauth.io/articles/identity-basics/multi-tenancy-vs-single-tenant-idaas-solutions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/multi-tenancy-vs-single-tenant-idaas-solutions</guid>
<description>Choosing between multi-tenant and single-tenant IDaaS solutions comes down to an organization's business objectives and requirements. Which trade-offs are you willing to make?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Open Source vs. Commercial Auth Providers</title>
<link>https://fusionauth.io/articles/identity-basics/open-source-vs-commercial</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/open-source-vs-commercial</guid>
<description>A look at the pros and cons of open-source and commercial authentication providers.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Why Outsource Your Auth System And How To Sell The Decision To Your Company</title>
<link>https://fusionauth.io/articles/identity-basics/outsource-auth-system-blueprint</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/outsource-auth-system-blueprint</guid>
<description>Why should you consider outsourcing your authentication, authorization and user management system, and how to get buy-in from your teammates.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>User-Centric Security - How Passkeys Improve User Experience</title>
<link>https://fusionauth.io/articles/identity-basics/passkeys-ux</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/passkeys-ux</guid>
<description>We discuss the history of passkeys, how developers use them, and how they improve the overall user experience (UX) of application login.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Is Passwordless Authentication Ready for Regulatory Compliance?</title>
<link>https://fusionauth.io/articles/identity-basics/passwordless-regulatory-compliance</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/passwordless-regulatory-compliance</guid>
<description>Passwordless authentication methods, like passkeys, are all the rage. Does passwordless have a home in highly-regulated industries such as banking, healthcare, and local government?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Best Practices For Registration Forms</title>
<link>https://fusionauth.io/articles/identity-basics/registration-best-practices</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/registration-best-practices</guid>
<description>Users have to fill out registration forms to access applications. How can you make this as painless as possible?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Slow Migration of User Data</title>
<link>https://fusionauth.io/articles/identity-basics/slow-migration</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/slow-migration</guid>
<description>What is a slow migration and when does it make sense to perform one?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Value of Trying Your Auth Provider Before You Commit</title>
<link>https://fusionauth.io/articles/identity-basics/try-before-you-buy</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/try-before-you-buy</guid>
<description>What are the benefits of trying an auth system provider before you integrate?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What Is Identity Proofing?</title>
<link>https://fusionauth.io/articles/identity-basics/what-is-identity-proofing</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/what-is-identity-proofing</guid>
<description>What is identity proofing and why might you need it?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What Is OpenID Connect (OIDC) and How It Works</title>
<link>https://fusionauth.io/articles/identity-basics/what-is-oidc</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/what-is-oidc</guid>
<description>Explore OpenID Connect (OIDC) and understand how it enhances authentication for web and mobile applications. Learn its workings, benefits, and more.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What Is SCIM, The System For Cross-Domain Identity Management?</title>
<link>https://fusionauth.io/articles/identity-basics/what-is-scim</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/what-is-scim</guid>
<description>What is SCIM, why would you use it, and how does it work?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What To Do When Your Auth System Vendor Gets Acquired</title>
<link>https://fusionauth.io/articles/identity-basics/what-to-do-when-auth-system-vendor-acquired</link>
<guid isPermaLink="true">https://fusionauth.io/articles/identity-basics/what-to-do-when-auth-system-vendor-acquired</guid>
<description>Authentication is an integral part of your application, and as such the acquisition of your auth vendor isn't like other acquisitions.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Authentication Workflows Overview</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/authentication-workflows-overview</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/authentication-workflows-overview</guid>
<description>An overview of the various types of login & authentication workflows used by applications.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Login & Authentication Workflow Articles</title>
<link>https://fusionauth.io/articles/login-authentication-workflows</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows</guid>
<description>A detailed explanation of the various types of login & authentication workflows used by applications.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Mobile Login To Backend With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/mobile/native-login-form-to-application-backend-jwts-refresh-tokens</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/mobile/native-login-form-to-application-backend-jwts-refresh-tokens</guid>
<description>An explanation of mobile application login using a native login form that submits to the application backend with JWTs and refresh tokens</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Mobile Login To FusionAuth With JWTs And Refresh Tokens - Recommended</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/mobile/native-login-form-to-fusionauth-jwts-refresh-tokens</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/mobile/native-login-form-to-fusionauth-jwts-refresh-tokens</guid>
<description>An explanation of mobile application login using a native login form that submits directly to FusionAuth with JWTs and refresh tokens</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Mobile OAuth Login Using Resource Owner Password Credentials Grant With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/mobile/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/mobile/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens</guid>
<description>An explanation of mobile application login using a native login form that submits to the application backend ( with JWTs and refresh tokens) which calls FusionAuth's OAuth Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To Backend With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-jwts-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits to the application backend with JWTs and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To Backend With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-sessions-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits to the application backend with server-side sessions plus refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To Backend With Sessions</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-application-backend-sessions</guid>
<description>A backend code explanation and example of single-page application login using a native login form that submits to the application backend with server-side sessions</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To FusionAuth With JWTs And Refresh Tokens In Local Storage</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-jwts-local-storage-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-jwts-local-storage-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits directly to FusionAuth with JWTs in local storage and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To FusionAuth With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-jwts-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits directly to FusionAuth with JWTs and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application Native Login To FusionAuth (Same Domain) With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-same-domain-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/native-login-form-to-fusionauth-same-domain-jwts-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits directly to FusionAuth on the same domain with server-side sessions with refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Authorization Code Grant With JWTs And Refresh Tokens - Recommended</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-jwts-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the authorization code grant with JWTs and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Authorization Code Grant With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-sessions-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the authorization code grant with server-side sessions plus refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Authorization Code Grant With Sessions - Recommended</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-authorization-code-grant-sessions</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the authorization code grant with server-side sessions</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Implicit Grant With JWTs In Cookies</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-jwts-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-jwts-cookies</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the implicit grant with JWTs in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Implicit Grant With JWTs In Local Storage</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-jwts-local-storage</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-jwts-local-storage</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the implicit grant with JWTs in local storage</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Implicit Grant With Session</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-implicit-grant-sessions</guid>
<description>An explanation of single-page application login using FusionAuth OAuth interface with the implicit grant with server-side sessions</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Resource Owner Password Credentials Grant With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits to the application backend (with JWTs and refresh tokens in cookies) which calls FusionAuth's Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Resource Owner Password Credentials Grant With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-sessions-refresh-tokens-cookies</guid>
<description>An explanation of single-page application login using a native login form that submits to the application backend (with server-side sessions plus refresh tokens in cookies) which calls FusionAuth's Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Single-Page Application OAuth Login Using Resource Owner Password Credentials Grant With Sessions</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/spa/oauth-resource-owner-password-credentials-grant-sessions</guid>
<description>An explanation of single-page application login using a native login form that submits to the application backend (with server-side sessions) which calls FusionAuth's Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp Native Login To Backend With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-jwts-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend with JWTs and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp Native Login To Backend With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-sessions-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend with server-side sessions plus refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp Native Login To Backend With Sessions</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/native-login-form-to-application-backend-sessions</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend with server-side sessions</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Authorization Code Grant With JWTs And Refresh Tokens - Recommended</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-jwts-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using FusionAuth OAuth interface with the authorization code grant with JWTs and refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Authorization Code Grant With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-sessions-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using FusionAuth OAuth interface with the authorization code grant with server-side sessions plus refresh tokens in cookies</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Authorization Code Grant With Sessions - Recommended</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-authorization-code-grant-sessions</guid>
<description>An explanation of webapp login using FusionAuth OAuth interface with the authorization code grant with server-side sessions</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Resource Owner Password Credentials Grant With JWTs And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-jwts-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend (with JWTs and refresh tokens in cookies) which calls FusionAuth's OAuth Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Resource Owner Password Credentials Grant With Sessions And Refresh Tokens</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-sessions-refresh-tokens-cookies</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-sessions-refresh-tokens-cookies</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend (with server-side sessions plus refresh tokens in cookies) which calls FusionAuth's Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>WebApp OAuth Login Using Resource Owner Password Credentials Grant With Sessions</title>
<link>https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-sessions</link>
<guid isPermaLink="true">https://fusionauth.io/articles/login-authentication-workflows/webapp/oauth-resource-owner-password-credentials-grant-sessions</guid>
<description>An explanation of webapp login using a native login form that submits to the application backend (with server-side sessions) which calls FusionAuth's OAuth Resource Owner's Password Grant</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Complete List of OAuth 2 Grants</title>
<link>https://fusionauth.io/articles/oauth/complete-list-oauth-grants</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/complete-list-oauth-grants</guid>
<description>Sure, you know the Authorization Code grant and the Client Credentials grant, but what other OAuth grants are there?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Differences Between OAuth 2 And OAuth 2.1</title>
<link>https://fusionauth.io/articles/oauth/differences-between-oauth-2-oauth-2-1</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/differences-between-oauth-2-oauth-2-1</guid>
<description>The OAuth 2.1 draft will soon be published as an RFC. Why did they write it? How were the OAuth 2 specifications modified? What will you have to change?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>OAuth Articles</title>
<link>https://fusionauth.io/articles/oauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth</guid>
<description>Learn the best practices and workflows for implementing OAuth for traditional web, single page, and mobile native applications.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>What is OAuth 2.0 and How does it Work?</title>
<link>https://fusionauth.io/articles/oauth/modern-guide-to-oauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/modern-guide-to-oauth</guid>
<description>Discover what OAuth 2.0 is, how it works, and its importance in securing and managing access to online resources. Learn about its key grants and benefits.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>OAuth Device Authorization</title>
<link>https://fusionauth.io/articles/oauth/oauth-device-authorization</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/oauth-device-authorization</guid>
<description>How to implement OAuth Device Authorization for set-top devices like Roku, AppleTV, and XBox and more.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Storing OAuth Tokens</title>
<link>https://fusionauth.io/articles/oauth/oauth-token-storage</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/oauth-token-storage</guid>
<description>OAuth grants return tokens, but what should you do with them?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>OAuth V1 Signed Requests</title>
<link>https://fusionauth.io/articles/oauth/oauth-v1-signed-requests</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/oauth-v1-signed-requests</guid>
<description>Twitter doesn't make it easy to build an OAuth v1 login with Java. Here's how we did it for FusionAuth.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>SAML vs. OAuth - Understand Differences and Benefits</title>
<link>https://fusionauth.io/articles/oauth/saml-vs-oauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/saml-vs-oauth</guid>
<description>SAML and OAuth are two systems that make secure user access more convenient and secure. Although they’re superficially similar, under the hood there are more differences than you might expect.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Value of Standards-Compliant Authentication</title>
<link>https://fusionauth.io/articles/oauth/value-standards-compliant-authentication</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/value-standards-compliant-authentication</guid>
<description>What are common auth standards and why should you consider using them in your application?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Why Is There No Auth In OAuth?</title>
<link>https://fusionauth.io/articles/oauth/why-no-authentication-in-oauth</link>
<guid isPermaLink="true">https://fusionauth.io/articles/oauth/why-no-authentication-in-oauth</guid>
<description>OAuth and OIDC are all about authentication, right? Why is there no process of authentication defined in the specifications?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Why Adaptive MFA is a Game Changer for User Experience</title>
<link>https://fusionauth.io/articles/security/adaptive-mfa-user-experience</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/adaptive-mfa-user-experience</guid>
<description>Users find traditional multi-factor authentication to be a burden. In this article, we examine how adaptive MFA can help improve the user experience (UX) rather than hinder it.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Exploring The Roles Of The Blue Team, The Red Team, And The Purple Team In Cybersecurity</title>
<link>https://fusionauth.io/articles/security/blue-team-red-team-purple-team</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/blue-team-red-team-purple-team</guid>
<description>Learn about the roles and some of the tools of the Blue Team and Red Team, as well as the concept of the Purple Team.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Breached Password Detection</title>
<link>https://fusionauth.io/articles/security/breached-password-detection</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/breached-password-detection</guid>
<description>What is breached password detection and why should you implement it.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Guide To User Data Security</title>
<link>https://fusionauth.io/articles/security/guide-to-user-data-security</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/guide-to-user-data-security</guid>
<description>Protect your user data with this step-by-step guide to secure your servers and applications.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Security Articles</title>
<link>https://fusionauth.io/articles/security</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security</guid>
<description>These resources are for the software developer who doesn't want to spend a lifetime wading through cryptographic algorithms and complicated explanations of arcane system administration topics to tackle software security.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>The Math of Password Hashing Algorithms And Entropy</title>
<link>https://fusionauth.io/articles/security/math-of-password-hashing-algorithms-entropy</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/math-of-password-hashing-algorithms-entropy</guid>
<description>Long passwords and one-way password hashing are standard strategies used to increase security and protect your user's passwords. Let's take a look at how they work and some new ideas to improve it.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Password Security Compliance Checklist</title>
<link>https://fusionauth.io/articles/security/password-security-compliance-checklist</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/password-security-compliance-checklist</guid>
<description>Use this combined checklist of leading password recommendations to strengthen your company's password security policy, meet compliance standards, and minimize the risk of data theft.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How SSO Makes for A More Personalized User Experience</title>
<link>https://fusionauth.io/articles/security/sso-ux</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/sso-ux</guid>
<description>This article explores various SSO scenarios and explains what benefits it brings to users and businesses in terms of security, user experience, and personalization.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>10 Steps To Secure Your Authentication System</title>
<link>https://fusionauth.io/articles/security/steps-secure-your-authentication-system</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/steps-secure-your-authentication-system</guid>
<description>This article will break down some of the essential steps developers need to take to maintain a secure authentication system.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Integrating Third-Party Services for Improved Identity Management</title>
<link>https://fusionauth.io/articles/security/third-party-services-ciam</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/third-party-services-ciam</guid>
<description>Using well-known and trusted identity providers means your visitors are more likely to sign up for your application because it's easy to do *and* they don't need to entrust you with their passwords.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>How to Implement Time Based One Time Passwords in FusionAuth</title>
<link>https://fusionauth.io/articles/security/time-based-one-time-passwords-totp</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/time-based-one-time-passwords-totp</guid>
<description>Discover how to implement time-based one-time passwords in FusionAuth. How are TOTP used? What does TOTP do?</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Auth Providers And The Zero Trust Architecture</title>
<link>https://fusionauth.io/articles/security/zero-trust-identity-provider</link>
<guid isPermaLink="true">https://fusionauth.io/articles/security/zero-trust-identity-provider</guid>
<description>Learn about the Zero Trust framework and how Identity Providers factor into it to protect your network and keep your resources secure.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Building A Secure Signed JWT</title>
<link>https://fusionauth.io/articles/tokens/building-a-secure-jwt</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens/building-a-secure-jwt</guid>
<description>How to correctly build and use a signed JSON Web Token.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Token Articles</title>
<link>https://fusionauth.io/articles/tokens</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens</guid>
<description>Everything you need to know to implement software tokens for secure authentication.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Components of JWTs Explained</title>
<link>https://fusionauth.io/articles/tokens/jwt-components-explained</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens/jwt-components-explained</guid>
<description>Technical article explains JSON Web Tokens (JWT), their component parts, and how they are used for authentication.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Pros And Cons of JWTs</title>
<link>https://fusionauth.io/articles/tokens/pros-and-cons-of-jwts</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens/pros-and-cons-of-jwts</guid>
<description>The pros and cons of JWTs and why you should (or shouldn't) use them.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Revoking JWTs & JWT Expiration</title>
<link>https://fusionauth.io/articles/tokens/revoking-jwts</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens/revoking-jwts</guid>
<description>How to use and revoke JWTs for effective and efficient authorization management. Examples, diagrams & more.</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
<item>
<title>Tokens At The Microservices Context Boundary</title>
<link>https://fusionauth.io/articles/tokens/tokens-microservices-boundaries</link>
<guid isPermaLink="true">https://fusionauth.io/articles/tokens/tokens-microservices-boundaries</guid>
<description>How to handle tokens at the boundary of your microservices</description>
<pubDate>Mon, 01 Jan 2024 00:00:00 GMT</pubDate>
</item>
</channel>
</rss>
```

Reply:
Awesome!